<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PARS Viewer</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="layout_bottom" stretch="4,1">
      <item>
       <widget class="QLabel" name="label_image">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : black</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_right">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolBox" name="toolbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_dataset">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>266</width>
             <height>557</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Dataset Settings</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_filepath">
              <property name="text">
               <string>Filepath</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="text_filepath">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>D:\root\data\pli\msa\MSA0309_s0536-0695_c_s05.h5</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <spacer name="spacer_dataset">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>245</width>
                <height>510</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_volume_rendering">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>266</width>
             <height>557</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Volume Rendering Settings</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_volume_rendering_enabled">
              <property name="text">
               <string>Enabled</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="checkbox_volume_rendering_enabled">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_scalars">
              <property name="text">
               <string>Scalars</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="combobox_scalars">
              <item>
               <property name="text">
                <string>transmittance</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>retardation</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ftle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ftle_forward</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ftle_backward</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <spacer name="spacer_volume_rendering">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>200</width>
                <height>491</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_particle_advection">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>271</width>
             <height>540</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Particle Advection Settings</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_particle_advection_enabled">
              <property name="text">
               <string>Enabled</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="checkbox_particle_advection_enabled">
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QLabel" name="label_load_balance">
              <property name="text">
               <string>Load Balance</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QCheckBox" name="checkbox_load_balancing_enabled">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_integrator">
              <property name="text">
               <string>Integrator</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="5">
             <widget class="QComboBox" name="combobox_integrator">
              <item>
               <property name="text">
                <string>euler</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>modified_midpoint</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>runge_kutta_4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>runge_kutta_cash_karp_54</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>runge_kutta_dormand_prince_5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>runge_kutta_fehlberg_78</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>adams_bashforth_2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>adams_bashforth_moulton_2</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_step_size">
              <property name="text">
               <string>Step Size</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="5">
             <widget class="QLineEdit" name="text_step_size">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>0.5</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_particle_advection_iterations">
              <property name="text">
               <string>Iterations</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="5">
             <widget class="QLineEdit" name="text_particle_advection_iterations">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_stride">
              <property name="text">
               <string>Stride</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QLineEdit" name="text_stride_x">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>32</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3" colspan="2">
             <widget class="QLineEdit" name="text_stride_y">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>32</string>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QLineEdit" name="text_stride_z">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>32</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_line_radius">
              <property name="text">
               <string>Line Radius</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="5">
             <widget class="QLineEdit" name="text_line_radius">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="6">
             <spacer name="spacer_particle_advection">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>200</width>
                <height>364</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_color_mapping">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>266</width>
             <height>557</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Color Mapping</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_mode">
              <property name="text">
               <string>Mode</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="combobox_mode">
              <item>
               <property name="text">
                <string>hsl_constant_s</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hsl_constant_l</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hsv_constant_s</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hsv_constant_l</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>rgb</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_parameter">
              <property name="text">
               <string>Parameter</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="text_parameter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>0.5</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <spacer name="spacer_color_mapping">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>484</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_ray_tracing">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>266</width>
             <height>557</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Ray Tracing Settings</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="1">
             <widget class="QLineEdit" name="text_position_x">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>500.0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_position">
              <property name="text">
               <string>Position</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="text_up_x">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="text_position_y">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>750.0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="3">
             <widget class="QLineEdit" name="text_ray_tracing_iterations">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="text_forward_z">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_up">
              <property name="text">
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="text_forward_x">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_forward">
              <property name="text">
               <string>Forward</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="text_up_y">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="text_position_z">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>-1250.0</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="4">
             <spacer name="spacer_ray_tracing">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>248</width>
                <height>415</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="text_up_z">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="text_forward_y">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_ray_tracing_iterations">
              <property name="text">
               <string>Iterations</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_update">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="layout_top">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_ip">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_ip">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_port">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_port">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_connect">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="connect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="disconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
